README.md

```diff
@@ -348,17 +359,24 @@ See [Custom SQLite builds](Documentation/CustomSQLiteBuilds.md) for the installa
 pod 'GRDB.swift'
 ```
 
-GRDB can be installed as a framework, or a static library.
-
-## Swift Package Manager
+### ⚠️ Important Note for CocoaPods installation
 
-The [Swift Package Manager](https://swift.org/package-manager/) automates the distribution of Swift code. To use GRDB with SPM, add a dependency to `https://github.com/groue/GRDB.swift.git`
+Due to an [issue](https://github.com/CocoaPods/CocoaPods/issues/11839) in CocoaPods, it is currently not possible to deploy new versions of GRDB to CocoaPods. The last version available on CocoaPods is 6.24.1. If you want to install later versions of GRDB using CocoaPods, please use one of the following workarounds:
 
-GRDB offers two libraries, `GRDB` and `GRDB-dynamic`. Pick only one. When in doubt, prefer `GRDB`. The `GRDB-dynamic` library can reveal useful if you are going to link it with multiple targets within your app and only wish to link to a shared, dynamic framework once. See [How to link a Swift Package as dynamic](https://forums.swift.org/t/how-to-link-a-swift-package-as-dynamic/32062) for more information.
+- Depend on the `GRDB6` branch. This is more or less equivalent to what `pod 'GRDB.swift', '~> 6.0'` would normally do, if CocoaPods would accept new GRDB versions to be published:
+```ruby
+# Can't use semantic versioning due to https://github.com/CocoaPods/CocoaPods/issues/11839
+pod 'GRDB.swift', git: 'https://github.com/groue/GRDB.swift.git', branch: 'GRDB6'
+```
+- Depend on a specific version explicitly (Replace the tag with the version you want to use):
+```ruby
+# Can't use semantic versioning due to https://github.com/CocoaPods/CocoaPods/issues/11839
+# Replace the tag with the tag that you want to use.
+pod 'GRDB.swift', git: 'https://github.com/groue/GRDB.swift.git', tag: 'v6.29.0' 
+```
+Once CocoaPods resolves this issue and allows deployment of new GRDB versions, users will be able to return to the standard installation method described at the beginning of this section.
```

Comment:
Please remove this sentence, which has not its place in public documentation. I would not be happy to read it if I were a CocoaPods maintainer.

Reply:
Removed